docs/user/staking/ethereum/lsd-pools/lido.md

```diff
@@ -0,0 +1,62 @@
+# Lido
+
+---
+
+### **Lido Overview**
+
+Lido is a liquid staking solution for Ethereum. It allows users to stake their ETH while participating in on-chain activities, such as lending, to compound returns. Lido provides a seamless and secure staking experience for users and **node operators**. A Node Operator contributes to this module by running infrastructure. The module was designed for solo stakers and community stakers, providing them with the unique ability of becoming Ethereum validator with way less than 32 ETH and one of the friendliest UX possible, but may be utilized by anyone, including professional operators.
+
+**Where dappnode comes in?** Dappnode provides an easy-to-use interface for setting up a node operator for Lido. The process involves creating keystores, setting up the infrastructure, uploading the keystores, and depositing to create your node operator.
+
+---
+
+### **Package Key Features**
+
+- **DAO Governance**: Governed by a DAO to ensure decentralization and security.
+- **Community Staking Module**: Enables the community to participate as node operators staking on behalf of the DAO with way less than the 32 ETH required for solo staking.
+
+---
+
+### **First Steps to create a Node Operator in Dappnode**
+
+1. **Create the Keystores**: Generate keystores for the desired number of validators (each keystore equals 32 ETH). The keystores must be created with the withdrawal credential of Lido:
```

Comment:
I wouldnt specify 32 ETH here, as the user will only have to provide 2 ETH + bond, Lido stakes the remaining ~28 ETH